### PR TITLE
Added support for Bert

### PR DESCRIPTION
**Pull request by jkrukowski**
In this PR I took some changes from https://github.com/huggingface/swift-transformers/pull/89 (which seems to be quite old and I'm not sure if the author is willing to continue). I've tried to achieve the same with more focused set of changes and I've added some tests.

Side note: if this is good to merge maybe it's ok to tag @ashvardanian as a co-author?

### CODE REVIEW

**Review by pcuenca — COMMENTED** [review]
Looks good to me, @jkrukowski!

Of course, let's add @ashvardanian as a co-author. Can you submit (or resubmit) your commit appending the following line to the commit description?

```
Co-authored-by: Ash Vardanian <1983160+ashvardanian@users.noreply.github.com>
```

(I believe it should work)

**Comment by jkrukowski** [comment]
@pcuenca done, I think it worked, thanks!

**Comment by piotrkowalczuk** [comment]
I'm glad it's coming. I have been working on the same thing and was about to submit a PR ;) 
- https://github.com/piotrkowalczuk/swift-transformers/commit/e240cda6dfde558a1bedcc74410d8c81f542897d
- https://github.com/piotrkowalczuk/swift-transformers/commit/15e43455c249aa213f8ed8d09fb70ec0d1821ddd

File: Tests/PreTokenizerTests/PreTokenizerTests.swift [hunk 1]
```diff
@@ -171,4 +171,24 @@ class PreTokenizerTests: XCTestCase {
...
+        )
+        XCTAssertEqual(
+            preTokenizer1.preTokenize(text: "   Hey,    friend ,    what's up?  "),
+            ["Hey", ",", "friend", ",", "what", "\'", "s", "up", "?"]
+        )
+        XCTAssertEqual(
+            preTokenizer1.preTokenize(text: "   Hey,    friend ,  0 99  what's up?  "),
+            ["Hey", ",", "friend", ",", "0", "99", "what", "\'", "s", "up", "?"]
+        )
+    }
```
Comment (piotrkowalczuk) [inline Tests/PreTokenizerTests/PreTokenizerTests.swift:193]: Original Rust implementation also had this test case scenario:

```
        XCTAssertEqual(
            preTokenizer.preTokenize(text: "野口里佳 Noguchi Rika"),
            ["野", "口", "里", "佳", "Noguchi", "Rika"]
        )
```

Comment (jkrukowski) [reply Tests/PreTokenizerTests/PreTokenizerTests.swift:193]: I noticed that as well, but figured that the architecture here is a bit different and it should be handled by `BertNormalizer`, is this assumption correct @pcuenca?

Comment (pcuenca) [reply Tests/PreTokenizerTests/PreTokenizerTests.swift:193]: Yes, I think this is great as it is, we can always iterate if needed.

**Comment by pcuenca** [comment]
Merging now, thanks again @jkrukowski, @ashvardanian and @piotrkowalczuk, very much appreciated! 🙌

**Comment by ashvardanian** [comment]
Glad it helped! Hopefully the other patches can also make it to the upstream if someone has the ambition to polish them 🤗

**Pull request merged**